index.html

```diff
@@ -6,58 +6,65 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Cabin|Roboto:400,700|Roboto+Condensed:400,700" rel="stylesheet">
+
 </head>
 
 <body>
-    <header style="text-align: center;">
+    <header>
         <h1>Timur Buchberger</h1>
         <nav>
-            <a href="#contact">Contact</a>
             <a href="#bio">Bio</a>
+            <a href="#contact">Contact</a>
         </nav>
     </header>
-    <figure>
-        <img src="img/Cat.jpeg"
-        alt="placeholder picture of a cute cat">
-        <br />
+    <div class="hero">Responsive Front End Development</div>
+    <!--<figure>
+        <img src="img/Cat.jpeg" alt="placeholder picture of a cute cat">
         <figcaption>A placeholder picture of a beautiful cat</figcaption>
```

Comment:
Usually we don't leave commented out code in and don't merge it into the main branch. That one should be as clean as possible 🙂

Reply:
Bad practice, hopefully it won't happen again 😃